flink-libraries/flink-table/src/test/scala/org/apache/flink/table/ExpressionReductionTest.scala

```diff
@@ -458,4 +459,52 @@ class ExpressionReductionTest extends TableTestBase {
     util.verifySql(sqlQuery, expected)
   }
 
+  // todo this NPE is caused by Calcite, it shall pass when [CALCITE-1860] is fixed
+  @Ignore
+  @Test(expected = classOf[NullPointerException])
```

Comment:
What is the expected result of the test given that CALCITE-1860 is fixed?
Since the test is ignored, we should implement the test such that is validates the expected result.